board/safety.h

```diff
@@ -518,3 +518,14 @@ bool steer_torque_cmd_checks(int desired_torque, int steer_req, const SteeringLi
 
   return violation;
 }
+
+void pcm_cruise_check(int cruise_engaged) {
```

Comment:
We can also create a global pcm_cruise boolean, and just check this in generic_rx_checks, even more similar to openpilot's carstate.

Reply:
this is a good start. we can see what makes sense after doing these obvious, copy/paste cleanups